frontend/src/hooks/useContractQueries.ts

```diff
@@ -98,42 +94,29 @@ export function useContractQueries() {
         data,
       });
       const result = await rpcClient.sendTransaction(signed);
+      const tx: TransactionItem = {
+        contractAddress: '',
+        localContractId: contract.value?.id ?? '',
+        hash: result,
+        type: 'deploy',
+        status: 'PENDING',
+        data: {},
+      };
 
-      if (result?.status === 'success') {
-        const tx: TransactionItem = {
-          contractAddress: result?.data.contract_address,
-          localContractId: contract.value?.id ?? '',
-          hash: result.data.transaction_hash,
-          type: 'deploy',
-          status: 'PENDING',
-          data: {},
-        };
-
-        notify({
-          title: 'Started deploying contract',
-          type: 'success',
-        });
-
-        trackEvent('deployed_contract', {
-          contract_name: contract.value?.name || '',
-        });
-
-        transactionsStore.clearTransactionsForContract(
-          contract.value?.id ?? '',
-        );
-        transactionsStore.addTransaction(tx);
-
-        return tx;
-      } else {
-        throw new Error(
-          typeof result?.message === 'string'
-            ? result.message
-            : 'Error Deploying the contract',
-        );
-      }
+      notify({
+        title: 'Started deploying contract',
+        type: 'success',
+      });
+
+      trackEvent('deployed_contract', {
+        contract_name: contract.value?.name || '',
+      });
+
+      transactionsStore.clearTransactionsForContract(contract.value?.id ?? '');
+      transactionsStore.addTransaction(tx);
+      return tx;
     } catch (error) {
       isDeploying.value = false;
-      console.error(error);
       notify({
```

Comment:
should we maybe keep this log? otherwise we lose the information about what happened

Reply:
you can see that now all errors are logged in the JsonRpcService :)

Reply:
I saw it further down :) great stuff